src/views/admin/PersonCreationView.vue

```diff
@@ -0,0 +1,332 @@
+<script setup lang="ts">
+  import { usePersonStore, type CreatedPerson, type PersonStore } from '@/stores/PersonStore'
+  import { onMounted, type Ref } from 'vue'
+  import { type Router, useRouter } from 'vue-router'
+  import { useDisplay } from 'vuetify'
+  import { type Composer, useI18n } from 'vue-i18n'
+  import { useForm, type TypedSchema, type BaseFieldProps } from 'vee-validate'
+  import { object, string } from 'yup'
+  import { toTypedSchema } from '@vee-validate/yup'
+  import SpshAlert from '@/components/alert/SpshAlert.vue'
+  import LayoutCard from '@/components/cards/LayoutCard.vue'
+  import PasswordOutput from '@/components/form/PasswordOutput.vue'
+
+  const router: Router = useRouter()
+  const personStore: PersonStore = usePersonStore()
+  const { smAndDown }: { smAndDown: Ref<boolean> } = useDisplay()
+  const { t }: Composer = useI18n({ useScope: 'global' })
+
+  const validationSchema: TypedSchema = toTypedSchema(
+    object({
+      selectedVorname: string()
+        .matches(/^[A-Za-z]*[A-Za-z][A-Za-z0-9- ]*$/, t('admin.person.rules.vorname.matches'))
```

Comment:
Ich finde keine Validierungsregeln im Ticket, aber so ist es zu restriktiv.
Zahlen im Namen sind erlaubt, aber keine Akzente. Vorname und Nachname haben unterschiedliche Regeln.

Reply:
Guter Punkt. Im Backend ist es wohl so, dass der Vorname nicht mit einer Zahl anfangen darf und mindestens 2 Zeichen lang sein muss. Für den Nachnamen gab es keine besonderen Regeln. Bisher sind also nur die Regeln implementiert, damit es technisch funktioniert. Ich setze die Regeln auch für den Nachnamen und bin dann für ein Folgeticket, in dem wir die Validierungsregeln konkretisieren und implementieren. Wie siehst du das?

Reply:
Nach Rücksprache: Validierung aus dem Workshopergebnis noch im Scope umsetzen: https://ticketsystem.dbildungscloud.de/browse/SPSH-446

Reply:
Done.